packages/terra-modal-manager/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "terra-modal-manager",
+  "main": "lib/index.js",
+  "version": "0.1.2",
+  "description": "A Redux-backed Container component that dynamically presents components in a Terra Modal",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-modal-manager",
+    "ModalManager",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra/issues"
+  },
+  "homepage": "https://github.com/cerner/terra#readme",
+  "devDependencies": {
+    "react-redux": "^5.0.4",
+    "redux": "^3.6.0",
+    "terra-app-delegate": "^0.x",
+    "terra-toolkit": "^0.5.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-redux": "^5.0.4",
+    "redux": "^3.6.0",
+    "terra-base": "^0.x",
+    "terra-app-delegate": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^0.x",
+    "terra-slide-group": "^0.x",
+    "terra-mixins": "^1.0.0",
+    "terra-modal": "0.3.0",
+    "terra-responsive-element": "^0.x"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "$(cd ..; npm bin)/rimraf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "$(cd ..; npm bin)/stylelint src/**/*.scss",
+    "props-table": "$(cd ..; npm bin)/props-table ./src/ModalManager.jsx --out-dir ./docs/props-table",
```

Comment:
In terra-core, we symlink to the props-table package. 

Add terra-props-table as a devDependency.
https://github.com/cerner/terra-core/blob/master/packages/terra-button/package.json#L42

Remove `$(cd ..; npm bin)` from the script.
https://github.com/cerner/terra-core/blob/master/packages/terra-button/package.json#L30